presto-main/etc/log.properties

```diff
@@ -9,3 +9,4 @@ io.prestosql=INFO
 com.sun.jersey.guice.spi.container.GuiceComponentProviderFactory=WARN
 com.ning.http.client=WARN
 io.prestosql.server.PluginManager=DEBUG
+org.apache.parquet.hadoop=WARN
```

Comment:
is this file used during tests?

I would log all logs `org.apache` at WARN level

Reply:
It is only used if you run the server manually in the IDE.

Reply:
Do product tests use these somehow?

Reply:
I'm not sure TBH

Reply:
This is actually used by `HIVE_TESTS=true` too (so `TestHiveClient` integration tests): https://travis-ci.com/prestosql/presto/jobs/173791261
but those tests are not problematic.